test/runtests.jl

```diff
@@ -1,5 +1,7 @@
 using LogExpFunctions
+using ChainRulesCore
```

Comment:
We already load ChainRulesTestUtils which also contains `frule`, so we can remove this import:
```suggestion
```